docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -562,6 +562,11 @@ Size of the index in <<byte-units, byte units>>.
 `translog`::
 <<index-modules-translog,Translog>> statistics.
 
+`dense_vector`::
```

Comment:
Nitpick: Could we make sure this is in alphabetical order?

Reply:
https://github.com/elastic/elasticsearch/pull/107962/commits/c4e6978bf20d448cb8f9942609164e2887bc0e23